extending/newtypes.po

```diff
@@ -46,13 +54,23 @@ msgid ""
 "that includes the fields you need and then change the values to suit your "
 "new type. ::"
 msgstr ""
+"Comme vous vous en doutez probablement maintenant, nous allons passer cela "
+"en revue et vous donner plus d'informations sur les différents "
+"gestionnaires. Nous n'allons pas suivre l'ordre dans lequel ils sont définis "
+"dans la structure, parce beaucoup d'informations préliminaires peuvent avoir "
+"un impact sur l'ordre des champs. Il est souvent plus facile de trouver un "
+"exemple qui inclut les champs dont vous avez besoin et de modifier les "
+"valeurs en fonction de votre nouveau type. ::"
 
 #: extending/newtypes.rst:33
 msgid ""
 "The name of the type -- as mentioned in the previous chapter, this will "
 "appear in various places, almost entirely for diagnostic purposes. Try to "
 "choose something that will be helpful in such a situation! ::"
 msgstr ""
+"Le nom du type -- comme mentionné dans le chapitre précédent, il apparaîtra "
+"à divers endroits, presque entièrement à des fins de diagnostic. Essayez de "
```

Comment:
Est-ce du deepl remagné ? "entièrement" me semble vraiment trop litéral ici, bien sûr que le nom sera utilisé en entier, mais presque *exclusivement* à des fins de diagnostic.

Reply:
Je traduirai : 
"almost entirely for diagnostic purposes."
par:
"dans la plupart des cas à des fins de diagnostic."

Ca me semble toujours un peu forcé, j'esseierai de l'améliorer ...